### DISCUSSION TITLE
Alexa integrazione in HA e sicurezza

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#380] [HUB: Home Assistant]
Ho letto che dopo aver seguito i passi della guida per l'integrazione di Alexa in Home Assistant è importante per la sicurezza cancellare il token di lunga durata attivato su Amazon console. E' vero ed è necessario? e soprattutto dopo funziona ancora l'integrazione?
ecco il passaggio da fare
````
- Mettere tutto in sicurezza -
Ora che tutto è in ordine potete ritornare sulla Funzione AWS Lambda
Scorrete sotto fino a "Variabili di ambiente"
rimuovete la riga LONG_LIVED_ACCESS_TOKEN
rimuovete (opzionalmente) la riga DEBUG
Scorrete in cima alla pagina e cliccate sul pulsante Salva
Infine ritornate sul vostro HA nella pagina dell'utente e cancellate il token di test`
```

**Comment by KC-inDomus** [comment]
beh, male non fa. però non dovrebbe esser tutto questo, problema, alla fine sul tuo account AWS ci entri solo tu (e Amazon, chiaro, ma allora lì il discorso si amplia a dismisura e in tutte le direzioni)

**Comment by calz64** [comment]
Grazie della risposta, in effetti parlare di privacy e sicurezza è lunga.
 Però se cancello il token su AWN, l'integrazione funziona lo stesso in locale? in sostanza smette solo l'allineamento dei dati sul cloud AWN?
